build-tools/scripts/javac-version

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+sysname=`uname`
+cmdname=`basename "$0"`
+exec javac -version 2>&1
```

Comment:
Note that on `macOS` it may invoke the default Java handler which will prompt the user to install Java. Not sure if it can be avoided, though.